server/src/main/antlr4/org/apache/iotdb/db/qp/strategy/SqlBase.g4

```diff
@@ -750,6 +754,8 @@ OPERATOR_LTE : '<=';
 
 OPERATOR_NEQ : '!=' | '<>';
 
+OPERATOR_IN : 'IN' | 'in' | 'In' | 'iN';
```

Comment:
This can be just `OPERATOR_IN : I N;`. The result is the same.

Reply:
Good suggestion. fixed.